src/backend/assets/js/tabs.js

```diff
@@ -26,6 +30,19 @@ tabs = function(options) {
       goToTab(index);
     });
   };
+  handleDisplay = function(className) {
+    className = className.split(' ')[0];
```

Comment:
This creates a global copy of className which can be accessed anywhere, `Use let className`